Assets/Character/Scripts/Run.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Run : MonoBehaviour {
+
+    public float moveSpeedX;
+    private float moveDirX;
```

Comment:
declare this in local function scope, not local class scope. Use smallest scope as possible